com.play0ad.zeroad.yaml

```diff
@@ -7,6 +7,8 @@ sdk-extensions:
   - org.freedesktop.Sdk.Extension.rust-stable
   - org.freedesktop.Sdk.Extension.llvm18
 command: 0ad
+env:
```

Comment:
This never worked for me. Hence the patch. Also it seems it broke the CI.

Reply:
Ah OK I just applied the change you had in another commit. I think I have found the issue, let me try..